src/components/Plugin/SafeSnap/CustomBlock.vue

```diff
@@ -6,25 +6,25 @@
     @submit="updateDetails"
   >
     <div v-if="infoLabel" class="mb-3 text-center">
-      <Label>
+      <UiLabel>
         {{ $t(infoLabel, [questionDetails.transactions.length]) }}
-      </Label>
+      </UiLabel>
     </div>
     <div v-if="showDecision" class="mb-3 link-color text-center">
-      <Label> {{ approvalData?.decision }} </Label>
+      <UiLabel> {{ approvalData?.decision }} </UiLabel>
```

Comment:
This should not be a <UiLabel> but a html <label>, you can change it to <b> to avoid confusion it will appear the same.

Reply:
Not sure what you mean here

Reply:
Oops just realize the `<>` text was stripped. I mean: This should not be a `<UiLabel>` but a html `<label>`, you can change it to `<b>` to avoid confusion it will appear the same.